src/applications/edu-benefits/0994/pages/benefitsEligibility.js

```diff
@@ -14,8 +15,8 @@ export const uiSchema = {
     'ui:title': '',
     'ui:description': eligibilityInstructions,
     'ui:options': {
-      expandUnder: 'appliedForVaEducationBenefits',
       expandUnderCondition: false,
```

Comment:
I think you can remove the expandUnderCondition, too

Reply:
@jbalboni Thanks for the recommendation, I've gone ahead and made that change.